bot/converters.py

```diff
@@ -259,3 +259,24 @@ class ISODateTime(Converter):
             dt = dt.replace(tzinfo=None)
 
         return dt
+
+
+class Literal:
+    """
+    Matches a literal, useful for e.g. flags.
+    Can be used with Optional.
+
+    Implementation by Danny#0007
+    """
+
+    def __class_getitem__(cls, item):
+        if not isinstance(item, tuple):
+            item = (item,)
+
+        class LiteralProxy(Converter):
+            @classmethod
+            async def convert(cls, ctx, argument):
+                if argument in item:
+                    return argument
+                raise BadArgument(f"Expected literal: one of {list(map(repr, self.literals))}")
+        return LiteralProxy
```

Comment:
Unnecessary to add at this point in time due to suggesting to change commands processing to the existing command groups format.